lib/python/test_file_classifier.py

```diff
@@ -263,6 +263,17 @@ def test_velocity(self):
         self.assertEqual(dest_dir, 'IMOS/ANMN/NRS/NRSROT/Velocity')
         self.assertEqual(dest_filename, filename)
 
+        filename = 'IMOS_ANMN-NRS_AETVZ_20150703T053000Z_NRSROT-ADCP_FV00_NRSROT-ADCP-1507-Workhorse-ADCP-43_END-20151023T034500Z_C-20151117T074309Z.nc'
+        testfile = os.path.join(self.tempdir, filename)
+        make_test_file(testfile, {'site_code':'NRSROT'},
+                       UCUR_MAG={},
+                       VCUR_MAG={},
+                       CSPD={}
+        )
```

Comment:
Yep, could do. I think this is also ok according to [PEP 8](https://www.python.org/dev/peps/pep-0008/#id17).